### DISCUSSION TITLE
When is the next release?

### DISCUSSION THREAD

**Discussion by CleanCut** [Leafwing-Studios/leafwing-input-manager#524] [Q&A]
Is the next release going to come when Bevy is next released, or sometime before?

I'm just trying to make some plans, and the unreleased changes are fairly extensive. If the release is going to come significantly before the next Bevy release, then I'll probably hop onto the main branch.

**Comment by alice-i-cecile** [comment]
No update until the next Bevy release.

We're going to be releasing a tiny update with a simple upgrade from the last released version for Bevy 0.14 compat, and then another one simultaneously with all of the changes from `main`.
